src/JsRoot/JsRoot.tsx

```diff
@@ -2,44 +2,49 @@ import Loader from "../Loader/Loader";
 import makeAsyncScriptLoader from "react-async-script";
 import React, { useEffect, useRef, useState } from 'react';
 
-const URL = `https://root.cern.ch/js/latest/scripts/JSRoot.core.js`;
-const globalName = "JSROOT";
+const urlJSROOT = 'https://root.cern.ch/js/latest/scripts/JSRoot.core.js';
+const keyJSROOT = 'JSROOT';
 
+declare global {
+    interface Window { [keyJSROOT]: any; }
+}
 
+const AsyncLoader = makeAsyncScriptLoader(urlJSROOT, { globalName: keyJSROOT })(Loader);
 
 function JsRoot() {
     const containerEl = useRef(null);
-    let [obj, setObj] = useState(undefined);
-    let [isDrawn, setIsDrawn] = useState(false);
-    let [scriptLoaded, setScriptLoaded] = useState(false);
-    let JSROOT:any;
-    const AsyncLoader = makeAsyncScriptLoader(URL,{globalName})(Loader);
-    
+    const [obj, setObj] = useState(undefined);
+    const [scriptLoaded, setScriptLoaded] = useState(false);
+
     const isLoaded = () => {
-        console.log((window as any).JSROOT);
-        JSROOT = (window as any).JSROOT;
-        JSROOT.openFile("./files/hsimple.root")
-            .then((file : any) => file.readObject("hpxpy;1"))
-            .then((obj : any) => {
-                setObj(obj) ;
-                setScriptLoaded(true);
-            });
+        const { JSROOT } = window;
+
+        // create example graph
+        let nbinsx = 20;
```

Comment:
```suggestion
        const nbinsx = 20;
```
I agree with deepsource